### ISSUE TITLE
Try Matomo

### ISSUE THREAD

**Issue by c-martinez** [fair-software/fair-software-frontend#72]
It would be good to have some statistics on site visits, but perhaps using an Open Source and privacy friendly solution such as https://matomo.org/ might be a good alternative to Google Analytics.

**Comment by ctwhome** [comment]
At the eScience Center we are using Matomo, maybe we can create an account with our own server? @ewan-escience @c-martinez

**Comment by c-martinez** [comment]
Yes, that would be great! But I guess we need to make a few changes on the site?
